### ISSUE TITLE
Create "project showcase" template

### ISSUE THREAD

**Issue by tdjames1** [RSE-Sheffield/RSE-Sheffield.github.io#830]
Should/could include:

* Project title
* Pretty picture
* Keywords (topics/technologies)
* Group/academic unit
* Quote from PI
* Short summary
* Longer description (for display on click through)
* Link to software
* Link to publication
* Link to project grant/funding award

Any other details?

**Comment by ns-rse** [comment]
Would a section for linking the project grant/funding award be overkill?

**Comment by ubdbra001** [comment]
> Would a section for linking the project grant/funding award be overkill?

Don't think so, potentially helps researchers who would be interested in including an RSE get an idea of where they can find money to do so

**Comment by EdwinB12** [comment]
Good idea Neil, think that can help rationalise the process for an academic

**Comment by EdwinB12** [comment]
@tdjames1 - Should we add something like "Picture"? Think we should try to encourage a photo for the project to make it look pretty (although not all projects lend themelves to picture I guess)

**Comment by ptheywood** [comment]
> @tdjames1 - Should we add something like "Picture"? Think we should try to encourage a photo for the project to make it look pretty (although not all projects lend themelves to picture I guess)

It's the second item in the original post: 

> * Pretty picture

**Comment by EdwinB12** [comment]
Oh bloody hell, I checked like 3 times before posting that, sorry!!

**Comment by tdjames1** [comment]
Only pretty pictures allowed :wink:

**Comment by ubdbra001** [comment]
Something like this?

```yaml
title: blah
rses: []
completed: true 
start_date: 29-10-2023
end_date: 29-10-2024
image: only-pretty.png
keywords: []
dept:
pi_name:
pi_quote: none
funding_sources: []
short_summary: "Lorem ipsum odor amet, consectetuer adipiscing elit. 
  Tempus fusce magnis cursus vitae aliquet mattis arcu.
  Eu conubia diam habitant aliquam platea sollicitudin.
  Viverra auctor vivamus blandit volutpat taciti ex quam nunc.
  Placerat duis porttitor, interdum donec sapien proin consequat vehicula magnis.
  Vulputate amet sed convallis viverra ad vehicula."
long_decription: "Lorem ipsum odor amet, consectetuer adipiscing elit.
  Porta neque sit facilisis curabitur maecenas interdum natoque donec vitae.
  Turpis ad urna nascetur ridiculus enim sociosqu.
  Potenti ipsum diam lectus suspendisse ligula?
  Mattis tortor tincidunt primis dignissim pretium dapibus leo ultricies tortor.
  Est in consectetur eleifend, habitasse fringilla id.
  Luctus facilisis aenean conubia penatibus sollicitudin urna.
  Alibero vivamus semper cursus ut ipsum.
  
  Justo porttitor conubia efficitur orci curabitur tempor; class tristique varius.
  Congue eros nisi non libero praesent malesuada.
  Justo leo lectus; magna ac magnis integer.
  Consectetur pellentesque tempor laoreet felis praesent lacus?
  Nisl urna enim vel euismod habitasse a.
  Libero quisque lectus dolor maecenas mattis quam malesuada.
  Rhoncus consequat pulvinar mauris faucibus ultrices sit in varius."
software_link: www.example.com
publication_links: []
```

One yaml file for each project/case study, and then use a loop and template to populate the page?

Edit: Updated with Edwin's suggestions.

**Comment by EdwinB12** [comment]
This looks excellent to me, guess RSE Involved too? And maybe a date?

**Comment by Robadob** [comment]
doi/publications could be useful in some cases.

**Comment by ubdbra001** [comment]
> This looks excellent to me, guess RSE Involved too? And maybe a date?

RSE(s) are a good shout.
What kind of date were you thinking: start and/or end?

**Comment by ubdbra001** [comment]
> doi/publications could be useful in some cases.

Space for this in the last item, set as an array in case there are multiple published works from a project.

**Comment by EdwinB12** [comment]
Both? This would give an idea of duration too. Guess it would have to support "Ongoing" or "present" too

**Comment by Robadob** [comment]
> > doi/publications could be useful in some cases.
> 
> Space for this in the last item, set as an array in case there are multiple published works from a project.

derp missed that, sorry

**Comment by ubdbra001** [comment]
Okay, updated the template.

**Comment by EdwinB12** [comment]
We should have a collaborate field too I think. Somewhere to put industry names, or other stakeholders :)

**Comment by ns-rse** [comment]
> We should have a collaborate field too I think. Somewhere to put industry names, or other stakeholders :)

Would that perhaps fall under the `funding_sources` field?

**Comment by EdwinB12** [comment]
I would argue funders and stakeholders can be very different? 

For a lot of our work, its just working with a PI, with one source of funding. But some have multiple PIs with multiple Stakeholders who are different entities to the funder. 

I think its also good to celebrate collaboration with external partners

**Comment by Robadob** [comment]
Funder sounds like a grant, collab is a person

**Comment by EdwinB12** [comment]
person or organisation?

**Comment by ns-rse** [comment]
My reasoning was that most Stakeholders external to the University will have skin in the game though and have provided funding, _ergo_ they could be listed as funders.

Projects could be done in collaboration with individuals from other Universities and funded by a separate body (or stakeholder!).

![image](https://github.com/user-attachments/assets/374516ee-777e-485e-b4e3-4b7f24c946b8)

**Comment by Robadob** [comment]
Funder: grant and/or company/organisation
Collab: people that actually worked on the project in some capacity (e.g. who would be named in publication), probably attributed to where they work if not Shef.

**Comment by tdjames1** [comment]
Could also list PI under collaborators to reduce number of fields to fill/display.

**Comment by ubdbra001** [comment]
#840: Draft PR with the draft projects page.
Will showcase at the team meeting.
